cms/migrations/0037_learning_outcomes_subhead_richtext.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.1.9 on 2019-07-03 09:59
+import re
+import wagtail.core.fields
+from django.db import migrations
+from django.db.models import F, Value
+from django.db.models.functions import Concat
+
+
+def wrap_in_paragraph(apps, schema_editor):
+    """
+    Wraps the CharField value with paragraph tags so the subsequent migration
+    to convert this field into a RichTextField will not impact any visual
+    display on client end.
+    """
+    if not schema_editor.connection.alias == "default":
+        return
+    LearningOutcomesPage = apps.get_model("cms", "LearningOutcomesPage")
+    LearningOutcomesPage.objects.update(
+        sub_heading=Concat(Value("<p>"), F("sub_heading"), Value("</p>"))
+    )
+
+
+def unwrap_paragraph_tag(apps, schema_editor):
+    """
+    Unwraps the value from paragraph tags so no they don't show up on the page once
+    it is rendered as plaintext.
+
+    Note: Django does not provide a database function to strip a pattern from a string
+    right within the query while referencing the model itself so we're doing this the
+    more cumbersome way.
+    """
+    if not schema_editor.connection.alias == "default":
+        return
+    LearningOutcomesPage = apps.get_model("cms", "LearningOutcomesPage")
+    for page in LearningOutcomesPage.objects.all():
+        page.sub_heading = re.sub("</p>$", "", re.sub("^<p>", "", page.sub_heading))
```

Comment:
We have [this utility method](https://github.com/mitodl/open-discussions/blob/master/open_discussions/utils.py#L166) in our open-discussions project to extract text from an HTML string. That seems like a much better solution here to me

Reply:
Sure, will update